src/pages/steps/AdditionalInformationStep/BookingInfoStep.tsx

```diff
@@ -436,7 +433,7 @@ const BookingInfoStep = ({
       delete bookingInfo.availabilityStarts;
     }
 
-    await addBookingInfoMutation.mutateAsync({
+    addBookingInfoMutation.mutateAsync({
```

Comment:
```suggestion
    addBookingInfoMutation.mutate({
```

there doesn't seem to be any real difference apart from `mutateAsync` returning a promise to be awaited

Reply:
👍